test/JDBC/expected/14_3__preparation__BABEL-3584-vu-prepare.out

```diff
@@ -0,0 +1,18 @@
+CREATE PROCEDURE get_tds_id_proc
```

Comment:
Test file names should not contain babel number, please replace it with get_tds_id